### ISSUE TITLE
Syncthing cannot load Ignore Patterns while #Include file is missing.

### ISSUE THREAD

**Issue by MickMorley** [syncthing/syncthing#8764]
### Include required information

Please be sure to include at least:

 - which version of Syncthing and what operating system you are using
   - 1.23.0
 - browser and version, if applicable
   - synctrazor/brave
 - what happened,
   - when accepting new folder share, after inserting Ignore Patterns the `.stignore` file is created automatically but in the GUI the Ignore Patterns show as empty, however, the file is not empty.  Syncthing is also not adhering to what is in `.stignore` and syncing all files.
 - what you expected to happen instead, and
   - Sync while respecting the Ignore Patterns in `.stignore`.  Maybe attempt to find the `#include` file first
 - any steps to reproduce the problem.
   - Yes, an example is below. Accept a new share from a remote pc.  Must use the `#include` function and make sure the `#include` file does not exist on host computer.
 - how to fix it
   - manually copy the `#include` file to the host computer and Syncthing will now load and use the Ignore Patterns. All `#include` files must exist

```
//global ignore file that contains unimportant extensions and files to ignore
#include .stignoresync

//exclude larger directories
bigfolder
biggerfolder
```

**Comment by calmh** [comment]
What's supposed to happen if the ignore patterns have a bad #include is that the patterns fail to load and the folder is stopped. This does happen in the normal case, if it doesn't when accepting a share then that's the bug, but "partially" loading an .stignore while ignoring #includes is not a thing we do or should do...

**Comment by MickMorley** [comment]
Understandable, and good to know it should have stopped.

Currently, it did not stop.  I tried the same process on 2 different PCs and it kept syncing. It just thinks the ignore patterns are empty.

**Comment by calmh** [comment]
Yeah the `#include` stuff is a red herring, the real issue here is that adding ignore patters as described doesn't actually take effect on the folder. No idea why yet.

**Comment by acffordyce973** [comment]
Came here to see if this was a known issue and report it if not. I've been trying out Syncthing with the hopes of moving over to it from Resilio. I have a workaround involving setting the pull order to "Smallest First", checking the box and adding my ignores, then doing "Pause All" followed by "Rescan All" once the #include files are synced. Maybe the ignores should be read and any #include files downloaded first? I'd figure this is possible since there are existing "Pull Order" options that must read the file data already.
